### PR TITLE
[13.0][FIX] stock_barcodes: Do not allow increase more than one unit in inventory lines with serial product tracking

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT41344

ping @carlosdauden @chienandalu

### CODE REVIEW

File: stock_barcodes/wizard/stock_barcodes_read_inventory.py [hunk 1]
```diff
@@ -65,10 +65,16 @@ def _add_inventory_line(self):
         StockInventoryLine = self.env["stock.inventory.line"]
         line = StockInventoryLine.search(self._prepare_inventory_line_domain(), limit=1)
         if line:
+            if self.product_id.tracking == "serial" and line.product_qty > 0.0:
+                self._set_messagge_info(
+                    "more_match", _("Inventory line with more than one quantity")
```
Comment (chienandalu) [inline stock_barcodes/wizard/stock_barcodes_read_inventory.py]: This is the message used in core:
```suggestion
                    "more_match", _("You can only process 1.0 %(product)s of products with unique serial number.") % {"product": self.product_id.name}
```

Comment (sergio-teruel) [reply stock_barcodes/wizard/stock_barcodes_read_inventory.py]: This message is too long

Comment (chienandalu) [reply stock_barcodes/wizard/stock_barcodes_read_inventory.py]: How about?:
```suggestion
                    "more_match", _("Inventory line with more than one unit in serial tracked product")
```

**Comment by sergio-teruel** [comment]
@chienandalu Changes done!!!

**Review by carlosdauden — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Pull request merged**